docs/migrating/migrating_from_ez_publish.md

```diff
@@ -43,11 +43,15 @@ If you are coming directly from legacy (4.x), you need to follow the best practi
     - And if you do this while on 5.x, you can use several of the [available legacy migration features](https://doc.ez.no/display/EZP/Legacy+code+and+features) to make the new code appear in legacy admin
 
 See Upgrade documentation on how to perform the actual upgrade: [Upgrade (eZ Publish Platform page)](https://doc.ez.no/display/EZP/Upgrade)
```

Comment:
Something is wrong with rendering and this line does now show in the output at all.